omod/src/main/java/org/openmrs/module/owa/web/controller/OwaManageController.java

```diff
@@ -56,7 +56,7 @@ public String deleteApp(@RequestParam("appName") String appName, ModelMap model)
 			appManager.deleteApp(appName);
 			model.clear();
 		}
-		return "redirect:manage.form";
+		return "manage.form";
```

Comment:
Have you looked at the effect of this change when one deletes an app from here? https://modules-refapp.openmrs.org/openmrs/module/owa/manage.form

Reply:
@dkayiwa I have updated the PR. An OWA can be deleted from https://modules-refapp.openmrs.org/openmrs/module/owa/manage.form.